packages/0xcert-ethereum-gateway/src/tests/core/gateway/actions-order/perform-instance-method.test.ts

```diff
@@ -208,6 +225,12 @@ spec.test('submits gateway fixed signed actions order to the network which execu
         receiverId: bob,
         abilities: [GeneralAssetLedgerAbility.CREATE_ASSET],
       },
+      {
+        kind: ActionsOrderActionKind.REVOKE_ASSET,
+        senderId: coinbase,
```

Comment:
We should migrate away from using "coinbase", this is outdated wording. Prefer "accounts[0]". Because light clients don't have a coinbase anymore.